web/src/dialog_widget.js

```diff
@@ -118,6 +119,29 @@ export function launch(conf) {
 
     const $submit_button = $dialog.find(".dialog_submit_button");
 
+    if (conf.update_submit_on_state_change) {
+        const originalValues = {};
```

Comment:
We generally don't use camelCase variable names and instead use names separated by underscores.